### ISSUE TITLE
Wilt drops spaces before colons in songs that contain them.

### ISSUE THREAD

**Issue by Hypsicratea** [ModalSeoul/Weeb.FM#30]
I am using RipFM and the spaces before the colons in two of my scrobbles were dropped when being pushed from Last.fm to Wilt, see image below.
![untitled](https://cloud.githubusercontent.com/assets/23017679/19819022/a26f5cf0-9d20-11e6-820e-026e7a926796.png)

**Comment by ModalSeoul** [comment]
Nothing in our code would do this. I'll run tests tonight when I get home, but I'm having a hard time thinking this is on our end.

Also, Last.fm shows something different than what they're actually storing a lot of the time. For instance, in some cases, they store songs/artists in all caps, even though that's not the correct way to write the song's title or the artist's name. But they present them in a proper way. If that makes any sense.

tl;dr I'll look at it but I think this one's on Last.

**Comment by Hypsicratea** [comment]
I think you are right about it being an issue with Last.fm because I did .fm in the Discord chat with that song and it showed up the same way it scrobbled to Wilt.

**Comment by ModalSeoul** [comment]
Aw, good to hear(kinda). 

I'll close this.
